pages/sign-up/sign-up.css

```diff
@@ -0,0 +1,52 @@
+body {
+    font-family: 'Roboto', sans-serif;
+    position: sticky;
+    background-color: var(--light-gray);
+}
+
+.main {
+    width: 20%;
+    margin: auto;
+    padding: 1rem 2rem;
+    border: var(--border-gray) solid 1.5px;
+    background-color: white;
+}
+
+.main-heading {
+    font-size: 1.5rem;
+    /* align-self: center; */
+    font-weight: bold;
+}
+
+.sign-button {
+    font-size: 1rem;
+    background-color: var(--primary-color);
+    padding: 0.5rem 1rem;
```

Comment:
0.5 rem can be converted to px but 1 rem I thought should stay the way it is, hence this would lead to inconsistency in unit in the same property, hence mentioned in rems. Pls do correct me if this does sound right.

Reply:
you can write 0.5rem as 8px and keep 1 rem as it is